### DISCUSSION TITLE
another way to crash an extension

### DISCUSSION THREAD

**Discussion by ILikeBypassing** [3kh0/ext-remover#237] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
so this is really (i mean super) easy to do
1. make a bookmark with your extensions page. for example - chrome-extension://ghlpmldmjjhmdgmneoaibbegkjjbonbk/pages/blockPage.html?

2. make a bookmark with the url - chrome://crash/. this is basically a bookmarklet
3. visit the extension page and click your crash bookmarklet
4. quickly visit your desired page ex: discord
5. spam this bookmarklet - javascript:(function(){window.onbeforeunload=function(){return '';}})();
6. boom you have another exploit to use while the devs are fixing this

please note that you only have ONE second to go to your desired site and remember to crash the extension before going to a redirect or a new site

**Comment by ILikeBypassing** [comment]
yes, i really like bypassing

**Reply by Theblackmamba824** [reply]
visual representation?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

**Reply by ILikeBypassing** [reply]
no what extension do u have

**Reply by Theblackmamba824** [reply]
lightspeed

**Reply by CheezeKake77** [reply]
How are you supposed to figure out your extension page? For example what is the goguardian extension page

**Reply by ILikeBypassing** [reply]
its in chrome://extensions

**Reply by Theblackmamba824** [reply]
> no what extension do u have

why do u ask

**Comment by DodoSeal** [comment]
How do I crash it if it doesn't have a page in it?

**Reply by ILikeBypassing** [reply]
go to your access denied page or your extensions blocked page

**Reply by DodoSeal** [reply]
It is a URL, not a `chrome-extension://ID_HERE`

**Reply by ILikeBypassing** [reply]
ohhhh ok then use the other method i posted. that one works for all extension

**Reply by DodoSeal** [reply]
Can you link me to it?

**Reply by ILikeBypassing** [reply]
https://github.com/3kh0/ext-remover/discussions/232
